hack/make/.ensure-frozen-images

```diff
@@ -4,7 +4,7 @@ set -e
 # this list should match roughly what's in the Dockerfile (minus the explicit image IDs, of course)
 images=(
 	busybox:latest
-	hello-world:latest
+	hello-world:frozen
```

Comment:
I love the idea, but this won't actually work because the last block of the `if` in this file is the "if all else fails, `docker pull <image>`", which will fail to pull the `frozen` tag here. :disappointed: